ldk/go/readme.md

```diff
@@ -11,9 +11,76 @@ go get -u github.com/open-olive/loop-development-kit/ldk/go
 Full documentation available on [pkg.go.dev](https://pkg.go.dev/github.com/open-olive/loop-development-kit/ldk/go).
 
 ### Loops
-This LDK can be used to write loops for Olive Helps. More detail about loops is available [here](docs/loops.md).
+Loops receive events and use them to generate relevant whispers. Loops choose which events they want to use and which they want to ignore.
 
-Example loops are available in the examples folder.
+This LDK can be used to write loops for Olive Helps. An example of the loop implementation is available [here](docs/loops.md).
+
+
+#### Interface
+
+Writing a loop boils down to writing an implementation for the `Loop` interface and serving it via `ldk.ServeLoopPlugin`.
+
+```go
+type Loop interface {
+	LoopStart(Sidekick) error
+	LoopStop() error
+}
+
+func main() {
+	...
```

Comment:
in place of the ellipsis, could you show them how we instantiate the logger, and maybe a placeholder for the Loop struct instantiation

Reply:
done